rd_ui/app/views/dashboard.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
                 </div>
             </div>
 
-            <div class="panel panel-default rd-widget-textbox" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
+            <div class="panel panel-default rd-widget-textbox" ng-style="widget.disableWidget" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
```

Comment:
I've just realized that this can be simplified by using the `ng-hide` directive like this: `<div class="..." ... ng-hide="widget.width == 0" ...>`. In which case you won't need the update to the filter nor the code in the controller.